nomenclature/definitions/subannual/README.md

```diff
@@ -43,6 +43,85 @@ with open(f'../subannual/months.yaml', 'r') as stream:
+Our convention makes reference to the following agreements:
+
+- Standard datetime format              : **UTC**  
+> The UTC datetime format is adopted to consider the following levels: day, hour, minute, and second if it is necessary (without considering time zones).
+> For example: `2020-01-01, 2020-01-02, ...` or `2020-01-01T13:00, ...`
+> The use of lists of dates and datetimes will be generated by the codes of the next section.
+
+- Hour duration                         :
+> It is understood that `2020-01-01T00:00` represents the duration from midnight until 1 a.m. (or whatever the next data point is in a multi-hour timestep resolution). For example: if we have a timestep equal to 2, the next data point will be 2 a.m.
+> The multi-hour timestep resolution is the total responsibility of the modeling team that must transform the data as part of the input/output processing workflow.
+
+- Calendar                              : **Use of real days**  
+> For any model with an hourly resolution, it assumes that uses 8760 hours for a common year and 8784 hours for leap years as 2020, 2024, etc.
+
+- Using time zone                       : **Everything is in UTC+01:00 (Central European Time)**
+> To emphasize that all-time data must be given in UTC format. And, do not include the special treatment for identifying time zones.
+
+- Summer/Winter-time in continuous-time format :
+> It is the total responsibility of each modelling team that must transform the data as part of the input/output processing workflow because the validate() function of nomenclature will validate how the timestamps are provided but it won't recognize the difference between UTC and local time as well as nor transform data from each other.
+
+#### Representatives timeslices:
+It refers to clustered hours and not consecutive that generates representatives timeslices. Some agreements are reflected in the following:
+
+- Using winter/summer time as representatives timeslices  :
+> To distinguish between different granularity levels of representative timeslices, It was proposed the following: `<Granularity>|<Name of timeslice>`. For example: `2 Season-2 Times|Summer-Day`.
+
+- Averaging values over the time span   :
+> A value is always the average for flow variables (i.e. It is the average between the subannual time and the subsequent one, where average is contingent on the lowest level of granularity - if you use 2020-01-01T13:00, it is hourly average, if you use 2020-01-01, it is daily average...).
+
+- Accumulating values over the time span:
+> A value at the start of the period for stock over the time-period until the start of the next timeslice.
+> For example, Capacity or Reservoir Level at 2020-01-01T13:00 is the value at 1pm, if subannual is given as 2020-01-01, it is understood as midnight that day, if it's January, it is understood as midnight on the first day of the month.
```

Comment:
If I'm getting your point, a 3rd bullet could be added to clarify if the value is divided automatically or avoid this assumption.  I prefer the second option, so we could fall into many details that can be managed before. I think that we should upload data after the aggregation/disaggregation process and try to avoid any automatic math operation.  Any opinion?